EasyRestProjectNetTeam2/EasyRestPages/ManageAdministratorPage.cs

```diff
@@ -0,0 +1,40 @@
+﻿using EasyRestProjectNetTeam2.Decorator;
+using EasyRestProjectNetTeam2.EasyRestComponentsObj;
+using OpenQA.Selenium;
+using SeleniumExtras.PageObjects;
+
+namespace EasyRestProjectNetTeam2.EasyRestPages
+{
+    public class ManageAdministratorPage : BasePage
+    {
+        public AddEmployeeComponent AddEmploeeComponent { get; private set; }
+
+        public ManageAdministratorPage(IWebDriver driver) : base(driver) { }
+
+        [FindsBy(How = How.XPath, Using = "//button[@title='Add Administrator']")]
+        private IWebElement _plusAdministratorButton;
+
+        [FindsBy(How = How.XPath, Using = "(//button[contains(@class, 'MuiButtonBase-root')])[2]")]
+        private IWebElement _deleteAdministratorButton;
+
+        [FindsBy(How = How.XPath, Using = "(//span[contains(@class, 'MuiTypography-root-41 MuiTypography-body1')])[5]")]
```

Comment:
I'm not sure that this locator would work. You have number 41, which is dynamic, I suppose.

Reply:
This locator works 'cause dynamic part is deleted[](url)
![d1](https://user-images.githubusercontent.com/103622690/184066268-0f2e2339-427e-4acc-98d4-4236c24fe156.png)
![d2](https://user-images.githubusercontent.com/103622690/184066271-52b31e5f-6269-4a66-92d0-cd811bf1f436.png)
But I made this shorter.
![Screenshot_20](https://user-images.githubusercontent.com/103622690/184066336-bee0257d-2e5c-4b86-b347-8ed85e4307e4.png)